ccc-server/src/main.rs

```diff
@@ -62,6 +87,25 @@ async fn error_handler(error: BoxError) -> impl IntoResponse {
 	}
 }
 
+fn init_tracing(tracing: LogStructure) {
+	if tracing == LogStructure::Default {
+		tracing_subscriber::fmt::init();
+	} else {
+		let output: Box<dyn Layer<tracing_subscriber::Registry> + Send + Sync> = match tracing {
+			LogStructure::Default => unimplemented!(),
+			LogStructure::Debug => layer().fmt_fields(format::Pretty::default()).boxed(),
+			LogStructure::Json => layer().json().boxed(),
+			LogStructure::Pretty => layer()
+				.event_format(format::Format::default().with_source_location(false))
+				.fmt_fields(format::PrettyFields::new())
+				.with_target(false)
+				.boxed(),
+		};
+
+		tracing_subscriber::registry().with(output).init();
```

Comment:
Separately, we can also support the `EnvFilter` that `RUST_LOG` configures with a layer in our tracing subscriber, so we could do things like

```sh
RUST_LOG=error cargo run --bin ccc-server
RUST_LOG=debug cargo run --bin ccc-server
RUST_LOG="warn,test::foo=info,test::foo::bar=debug" cargo run --bin ccc-server
```

adding env-filter trait
```toml
tracing-subscriber = { version = "0.3.17", features = ["json", "env-filter"] }
```

using `EnvFilter`
```rust
use tracing_subscriber::filter

let env_filter = tracing_subscriber::EnvFilter::from_default_env();

if tracing == LogStructure::Default {
	tracing_subscriber::fmt()
	.with_env_filter(env_filter)
	.init();
} else {
	tracing_subscriber::registry()
	.with(env_filter)
	.with(output)
	.init();
}
```

Reply:
Implemented in #104